Source/Libraries/openXDA.DataPusher/DataPusherEngine.cs

```diff
@@ -111,6 +112,8 @@ public DataPusherEngine(DataPusherSettings settings)
 
         public static event EventHandler<EventArgs<string, string, int>> UpdateProgressForMeter;
 
+        private static readonly string ConnectionString = "systemSettings";
```

Comment:
If `DataPusherEngine` has a `ConnectionFactory`, then why does it need to use this hardcoded settings category?

Reply:
You are correct that this is unneeded, it has been removed.